open-metadata-implementation/access-services/analytics-modeling/analytics-modeling-server/src/main/java/org/odpi/openmetadata/accessservices/analyticsmodeling/auditlog/AnalyticsModelingAuditCode.java

```diff
@@ -0,0 +1,120 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/* Copyright Contributors to the ODPi Egeria project. */
+package org.odpi.openmetadata.accessservices.analyticsmodeling.auditlog;
+
+
+import org.odpi.openmetadata.frameworks.auditlog.messagesets.AuditLogMessageDefinition;
+import org.odpi.openmetadata.frameworks.auditlog.messagesets.AuditLogMessageSet;
+import org.odpi.openmetadata.frameworks.auditlog.messagesets.MessageDefinition;
+import org.odpi.openmetadata.repositoryservices.auditlog.OMRSAuditLogRecordSeverity;
+
+/**
+ * The AnalyticsModelingAuditCode is used to define the message content for the OMRS Audit Log.
+ * <p>
+ * The 5 fields in the enum are:
+ * <ul>
+ * <li>Log Message Id - to uniquely identify the message</li>
+ * <li>Severity - is this an event, decision, action, error or exception</li>
+ * <li>Log Message Text - includes placeholder to allow additional values to be captured</li>
+ * <li>Additional Information - further parameters and data relating to the audit message (optional)</li>
+ * <li>SystemAction - describes the result of the situation</li>
+ * <li>UserAction - describes how a user should correct the situation</li>
+ * </ul>
+ */
+public enum AnalyticsModelingAuditCode implements AuditLogMessageSet {
+
+    SERVICE_INITIALIZING("OMAS-ANALYTICS-MODELING-0001",
+            OMRSAuditLogRecordSeverity.STARTUP,
```

Comment:
These message descriptions will not pass the audit log unit test.  It is ok for an initial drop but needs to be improved - look at unit test for the Audit log and exceptions in Asset Owner OMAS